src/api/app/jobs/report_to_scm_job.rb

```diff
@@ -0,0 +1,27 @@
+# We don't properly capitalize SCM in the class name since CreateJob is doing `CLASS_NAME.to_s.camelize.safe_constantize`
+class ReportToScmJob < CreateJob
+  ALLOWED_EVENTS = ['Event::BuildFail', 'Event::BuildSuccess'].freeze
+
+  queue_as :scm
+
+  def perform(event_id)
+    event = Event::Base.find(event_id)
```

Comment:
This previous comment from @hennevogel still stands:

> It will never come this far. 
> 
> ```ruby
> Event::Base.find(12345)
> ActiveRecord::RecordNotFound: Couldn't find Event::Base with 'id'=12345
> ```

I suggest changing the line to this:
```suggestion
    event = Event::Base.find_by(id: event_id)
```

Reply:
True

Reply:
Hm, I've been checking this and we'll be hitting [this after_perform](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/jobs/create_job.rb#L8) if we pass an id from a non-existent event. Thinking about alternatives, it's better to let it raise an error and detect this error in Errbit, so we can fix it outside this class

Reply:
Yes, since there isn't a way around it without affecting all jobs inheriting from `CreateJob`, we'll leave it as it is. This is also an edge case. The event should always be in the database.